### PR TITLE
Update windows-autopatch-register-devices.md

### PR DESCRIPTION
**Pull request by rlianmsft**
Making it clear on which methods can be used to populate the Windows Autopatch Device Registration group.

## Why

- Closes #[Issue Number]

### CODE REVIEW

File: windows/deployment/windows-autopatch/deploy/windows-autopatch-register-devices.md [hunk 1]
```diff
@@ -28,7 +28,13 @@ Windows Autopatch can take over software update management control of devices th
 
 ### About the use of an Azure AD group to register devices
 
-You must choose what devices to manage with Windows Autopatch by either adding them through direct membership or by nesting other Azure AD dynamic/assigned groups into the **Windows Autopatch Device Registration** Azure AD assigned group. Windows Autopatch automatically runs its discover devices function every hour to discover new devices added to this group. Once new devices are discovered, Windows Autopatch attempts to register these devices.
+You must choose what devices to manage with Windows Autopatch by adding them into the **Windows Autopatch Device Registration** Azure AD assigned group. Devices can be added using the following methods:
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopatch/deploy/windows-autopatch-register-devices.md:31]:
```suggestion
You must choose what devices to manage with Windows Autopatch by adding them to the **Windows Autopatch Device Registration** Azure AD assigned group. Devices can be added using the following methods:
```

File: windows/deployment/windows-autopatch/deploy/windows-autopatch-register-devices.md [hunk 2]
```diff
@@ -28,7 +28,13 @@ Windows Autopatch can take over software update management control of devices th
...
+
+- Direct Membership
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopatch/deploy/windows-autopatch-register-devices.md:33]:
```suggestion
- Direct membership 
```

File: windows/deployment/windows-autopatch/deploy/windows-autopatch-register-devices.md [hunk 3]
```diff
@@ -28,7 +28,13 @@ Windows Autopatch can take over software update management control of devices th
...
 ### About the use of an Azure AD group to register devices
 
-You must choose what devices to manage with Windows Autopatch by either adding them through direct membership or by nesting other Azure AD dynamic/assigned groups into the **Windows Autopatch Device Registration** Azure AD assigned group. Windows Autopatch automatically runs its discover devices function every hour to discover new devices added to this group. Once new devices are discovered, Windows Autopatch attempts to register these devices.
+You must choose what devices to manage with Windows Autopatch by adding them into the **Windows Autopatch Device Registration** Azure AD assigned group. Devices can be added using the following methods:
+
+- Direct Membership 
+- Nesting other Azure AD dynamic/assigned groups
+- Bulk operations – Import members 
+
+Windows Autopatch automatically runs its discover devices function every hour to discover new devices added to this group. Once new devices are discovered, Windows Autopatch attempts to register these devices.
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopatch/deploy/windows-autopatch-register-devices.md:37]:
```suggestion
Windows Autopatch automatically runs its discover devices function every hour to discover new devices added to this group. Once new devices are discovered, Windows Autopatch attempts to register these devices.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rlianmsft, kindly commit the minor suggested changes. Many thanks.

**Comment by rlianmsft** [comment]
@JohanFreelancer9 I think that is all of them committed. Let me know if not.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tiaraquan** [comment]
#sign-off

**Pull request merged**